ensdomains/ens.domains
# fix: Update Mailchain Case & Add New Logo

Name: Mailchain
Type: App
Level of integration: Resolving ENS names (forward and reverse lookup)
How to test:
1. [Install Mailchain](https://docs.mailchain.xyz/getting-started), compose a message, add ENS name to the `To` field.
2. When ENS is configured for a sender address, this will show in the `Addresses` section (see screenshot)
3. When a message is received from an address with ENS configured, this will be displayed in the messages section (see screenshot)

![image](https://user-images.githubusercontent.com/2022378/69500405-ae750300-0ef2-11ea-9ebd-5b03de04287b.png)
